### PR TITLE
5.2 hash program

### PR DESCRIPTION
**Pull request by erberger22**

### CODE REVIEW

**Comment by erberger22** [comment]
@bontgoy

**Comment by Bontgoy** [comment]
@erberger22 Is there anything specifically you have questions about that you would like me to review?  Please leave some questions if you're tagging me on PRs that are not part of the code review requests assignments.  I'd be happy to take a look and give feedback.

**Comment by erberger22** [comment]
I guess I didn't have any specific questions, just wondering if there was
any way I could improve my code readability. I've been given feedback that
my code is not very reader friendly, but no concrete ways to improve it.
I'll be sure to include questions next time I tag you.

File: ruby/hashes.rb [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+client ={
+	name: nil, 
+	age: nil,
+	num_child: nil,
+	theme: nil,
+	allergies: ["none"],
+	payment_method: nil
+}
+	def allergies_question 
+		user_answer="nil"
```
Comment (Bontgoy) [inline ruby/hashes.rb:23]: There are a lot of extra tabs here.  One tab in per nested level is the best practice.

File: ruby/hashes.rb [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+client ={
+	name: nil, 
+	age: nil,
+	num_child: nil,
+	theme: nil,
+	allergies: ["none"],
+	payment_method: nil
+}
+	def allergies_question
```
Comment (Bontgoy) [inline ruby/hashes.rb:22]: Tabbing inward is only for nesting.  If your commands or statements are not nested within something then place them all the way against the right side of the doc.

File: ruby/hashes.rb [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+		end
+	puts "How will services be payed for? Cash, check, credit, or barter"
+	client[:payment_method]=gets.chomp
+
+p client
+
+puts "Do any of these answers need to be changed? If yes enter the field that needs changing, if no enter 'none'"
+double_check=gets.chomp
+	if double_check=="none"
+	puts "Thank you for entering all that info!"
```
Comment (Bontgoy) [inline ruby/hashes.rb:57]: You should be nesting your code inside of statements that they depend on tab these inward.

**Review by Bontgoy — COMMENTED** [review]
Thanks for clarifying.

I left some comments for you.  I also use the [Airbnb Ruby Style Guide](https://github.com/airbnb/ruby) when I am wondering about my syntax.

I hope this helps.

**Pull request merged**